### PR TITLE
Don't loop when generating a single language graph

### PR DESCRIPTION
**Pull request by derickr**

### CODE REVIEW

**Comment by jimwins** [comment]
Whoops, yeah.

File: www/img-status-lang.php [hunk 1]
```diff
@@ -12,13 +12,13 @@
 
 $available_langs = revcheck_available_languages($idx);
 
+$lang = $_GET['lang'];
 $langs = array_keys($LANGUAGES);
```
Comment (kamil-tekiela) [inline www/img-status-lang.php:16]: Where is this used?

Comment (jimwins) [reply www/img-status-lang.php:16]: In the `in_array()` test and (possibly) call to `generate_image()` that immediately follows?

Comment (kamil-tekiela) [reply www/img-status-lang.php:16]: I can't see any usage of this variable in this file. Is the file included somewhere else? If not, then this variable is dead code.

Comment (kamil-tekiela) [reply www/img-status-lang.php:16]: Searching GitHub also confirms that this assignment can be deleted https://github.com/search?q=repo%3Aphp%2Fweb-doc%20%24langs&type=code

**Pull request merged**